roles/openshift_control_plane/tasks/registry_auth.yml

```diff
@@ -19,3 +19,22 @@
   retries: 3
   delay: 5
   until: master_oreg_auth_credentials_create is succeeded
+
+- name: Create credentials for any additional registries
+  docker_creds:
+    path: "{{ docker_cli_auth_config_path }}"
+    registry: "{{ item.host }}"
+    username: "{{ item.user | default('openshift') }}"
+    password: "{{ item.password }}"
+    # Test that we can actually connect with provided info
+    test_login: "{{ oreg_test_login | default(True) }}"
```

Comment:
Might want fine-grained control for test_login in this dictionary because skopeo will probably fail on self-signed cert registries; not sure if skopeo respects docker insecure registries, need to confirm.

Reply:
I'll add both test_login and tls_verify options. Defaulting both to true.